src/project.rs

```diff
@@ -9,6 +9,45 @@ pub struct ProjectInfo {
 	pub matrix_room_id: Option<String>,
 }
 
+#[derive(Default, Debug, Clone, Copy)]
+pub struct AuthorInfo {
+	pub is_owner: bool,
+	pub is_reviewer: bool,
```

Comment:
Should be `is_delegated_reviewer` or `is_delegated`, since it is distinct from a normal reviewer